### DISCUSSION TITLE
mpd service has no logging

### DISCUSSION THREAD

**Discussion by mqus** [termux/termux-packages#9365] [Q&A] [answered]
Afaict I can't get any logs from the mpd service because it is started with `--stdout` and no logger is set up. Where can I find those logs? Did I do something wrong? Is this intentional and why?

**Comment by Lodger-test** [comment]
Same thing. If mpd is started as
`sv start mpd`
it ignores log, set in mpd.conf

If mpd is sarted manualy, it uses it's log

**Comment by sylirre** [comment, marked as answer]
In `$PREFIX/var/log/sv/mpd/current`. Logs from stdout/stderr are captured by `svlogd` daemon started by termux-services.
